packages/wrangler/src/__tests__/delete.test.ts

```diff
@@ -197,34 +217,47 @@ function mockDeleteWorkerRequest(
 	} = {}
 ) {
 	const { env, legacyEnv, name } = options;
-	setMockResponse(
-		// there's no special handling for environments yet
-		"/accounts/:accountId/workers/services/:scriptName",
-		"DELETE",
-		async ([_url, accountId, scriptName], { method }, queryParams) => {
-			expect(accountId).toEqual("some-account-id");
-			expect(method).toEqual("DELETE");
-			expect(scriptName).toEqual(
-				legacyEnv && env
-					? `${name || "test-name"}-${env}`
-					: `${name || "test-name"}`
-			);
+	msw.use(
+		rest.delete(
+			"*/accounts/:accountId/workers/services/:scriptName",
+			(req, res, ctx) => {
+				expect(req.params.accountId).toEqual("some-account-id");
+				expect(req.params.scriptName).toEqual(
+					legacyEnv && env
+						? `${name || "test-name"}-${env}`
```

Comment:
```suggestion
						? `${name ?? "test-name"}-${env}`
```

Reply:
This was previously existing, but I can make the change.